core/src/main/java/com/linecorp/armeria/internal/server/annotation/ProcessedDocumentationHelper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.server.annotation;
+
+/**
+ * Helper class for Documentation processing.
+ */
+public final class ProcessedDocumentationHelper {
+    private ProcessedDocumentationHelper() {}
```

Comment:
Do you use the `@UtilityClass` lombok annotation? then it would remove all the boiler plate (no static needed for class members, no final needed for class, no private constructor needed for class)

Reply:
Nope, we don't use Lombok so that Armeria values performance and provides optimized code.

Reply:
Related issue: #2419